src/go/rpk/pkg/config/weak.go

```diff
@@ -351,13 +351,19 @@ func (rpc *RedpandaNodeConfig) UnmarshalYAML(n *yaml.Node) error {
 	if err := n.Decode(&internal); err != nil {
 		return err
 	}
+
+	// redpanda won't recognize rpc_server_tls if is a list.
+	v := reflect.ValueOf(internal.Other["rpc_server_tls"])
+	if v.Kind() == reflect.Slice {
+		fmt.Fprintf(os.Stderr, "WARNING: redpanda.yaml: redpanda.rpc_server_tls property is a list, redpanda only accepts single elements in this field")
```

Comment:
`WARNING: redpanda.yaml's redpanda.rpc_server_tls property is a list, and redpanda reads this as a single element. We cannot automatically fix this because it may break a running cluster. Please check our documentation for how to fix this on live clusters.`

Very verbose, also we need a guide for how to fix this (so, what happens when we turn off the entire cluster, that kind of thing)